CMakeLists.txt

```diff
@@ -62,6 +62,12 @@ function(trusted_firmware_build)
   set(VENEERS_FILE ${TFM_BINARY_DIR}/secure_fw/s_veneers.o)
   set(PSA_API_NS_PATH ${TFM_BINARY_DIR}/interface/libpsa_api_ns.a)
 
+  set(BL2_HEX_FILE ${CMAKE_BINARY_DIR}/tfm/bin/bl2.hex)
+  set_property(GLOBAL PROPERTY
+    bl2_PM_HEX_FILE
```

Comment:
@oyvindronningstad what is bl2_PM_HEX_FILE needed for?

Reply:
It's NCS-specific. We can probably add it from there instead, and remove it from here. I meant to remove it, just forgot.

Reply:
Yes, please, clean this PR up so it is not specific to the Nordic SDK :)

Reply:
Done